subtree_imports.go

```diff
@@ -5,6 +5,7 @@ package main
 
 import _ "github.com/taskcluster/taskcluster-cli/apis"
 import _ "github.com/taskcluster/taskcluster-cli/config"
+import _ "github.com/taskcluster/taskcluster-cli/config-cmd" // commands for config
```

Comment:
I still would advocate to move all the commands into a `cmds` subpackage, so the config commands would be in the `github.com/taskcluster/taskcluster-cli/cmds/config` package. I believe that it would make the layout and separation cleaner.